code/__DEFINES/maths.dm

```diff
@@ -96,6 +96,9 @@
 // Returns the nth root of x.
 #define ROOT(n, x) ((x) ** (1 / (n)))
 
+// Real modulus that handles decimals
```

Comment:
???

Reply:
Ask Goof about that. It was in the original PR.